neuralmonkey/decoders/sequence_classifier.py

```diff
@@ -23,18 +23,33 @@ def __init__(self,
                  encoders: List[Any],
                  vocabulary: Vocabulary,
                  data_id: str,
-                 layers: Optional[List[int]]=None,
-                 activation: Callable[[tf.Tensor], tf.Tensor]=tf.tanh,
+                 layers: List[int]=None,
+                 activation_fn: Callable[[tf.Tensor], tf.Tensor]=tf.nn.relu,
```

Comment:
no puvodne jsem jej chtel zmenit i v nelinearni projekci, ale tu jsem prave videl ze se pouziva na plno mistech, tak jsem jen vlozil koment, ze by to stalo za to dat jinej default (preci jen relu docela valcuje tanh, napric ruznymi clanky). Vic by k tomu mozna vedel @foxik :)